src/components/Layout/Navbar.vue

```diff
@@ -28,7 +28,7 @@
       </v-menu>
       <template v-if="($vuetify.breakpoint.width >= 600) && !isLoggedIn">
         <v-btn
-          id="nav-login"
+          data-test-id="nav-login"
```

Comment:
While it's good to use the `data-test-id`s for testing, I'm not sure it's safe to remove existing `id` attributes. We'd better leave them in here so code that expects them to exist (in case it exists?) keeps finding them.